src/redux-store/reducers/user-reducer.js

```diff
@@ -0,0 +1,35 @@
+import * as types from "../actions/actionTypes";
+
+const initialLoginState = {
+  username: "",
+  password: ""
+};
+
+export const loginReducer = (state = initialLoginState, action) => {
+  switch (action.type) {
+    case types.LOGIN_START:
```

Comment:
You seem to only be tracking the LOGIN_START, and it doesn't seem to have any effect on state. You need to have cases for it all the way through to failure and each case has to affect a state somehow (maybe like switching to logging in true and then to false or whatever) - that's the whole point of. That way you can render things in the button like "logging in" until the backend successfully approves it.